### PR TITLE
Add back Japanese subdomain

### PR DESCRIPTION
**Pull request by nar1n**

### CODE REVIEW

File: multisrc/overrides/wpcomics/mangatoro/src/MangaToro.kt [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+class MangaToroFactory : SourceFactory {
+    override fun createSources(): List<Source> = listOf(
+        MangaToro(),
+        MangaToroRAW(),
+    )
+}
+class MangaToroRAW : WPComics("MangaToro RAW", "https://ja.mangatoro.com", "ja") {}
+
+class MangaToro : WPComics("MangaToro", "https://mangatoro.com", "en") {}
```
Comment (Riztard) [inline multisrc/overrides/wpcomics/mangatoro/src/MangaToro.kt:14]: remove the empty brackets?

**Comment by Riztard** [comment]
and conflict

**Comment by nar1n** [comment]
sed need to fetch upstream again

**Comment by Riztard** [comment]
or just fix the conflict

**Pull request closed without merging**